contrib/standalone_buffer/Makefile

```diff
@@ -50,6 +50,9 @@ $(SETUPROOT)/BufferExpander.o: $(SETUPROOT)/modmesh/buffer/BufferExpander.cpp Ma
 $(SETUPROOT)/SimpleArray.o: $(SETUPROOT)/modmesh/buffer/SimpleArray.cpp Makefile
 	c++ $(CFLAGS) -c $< -o $@
 
+$(SETUPROOT)/neon_wrap.o: $(SETUPROOT)/modmesh/simd/neon/neon_wrap.cpp Makefile
```

Comment:
Thank you for taking care of the special build.  I forgot to mention it in the first place.